cmd/soroban-rpc/internal/events/cursor_test.go

```diff
@@ -0,0 +1,84 @@
+package events
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
As you can see, github actions generated the following warning:
`File is not `goimports`-ed with -local github.com/golangci/golangci-lint (goimports)`

to work around it, just move the `github.com/stretchr/testify/assert` to the bottom.